Description: Fix undefined behavior in VfrCompiler.
Origin: http://sourceforge.net/p/edk2-buildtools/code/2667/
Bug-Debian: http://bugs.debian.org/773492
Author: Reza Jelveh <reza.jelveh@tuhh.de>
Reviewed-by: Eric Dong <eric.dong@intel.com>
Last-Update: 2014-12-18
Applied-Upstream: commit:r2667

Index: edk2/BaseTools/Source/C/VfrCompile/VfrCompiler.cpp
===================================================================
--- edk2.orig/BaseTools/Source/C/VfrCompile/VfrCompiler.cpp
+++ edk2/BaseTools/Source/C/VfrCompile/VfrCompiler.cpp
@@ -378,6 +378,8 @@
 
   SET_RUN_STATUS (STATUS_STARTED);
 
+  SET_RUN_STATUS (STATUS_STARTED);
+
   OptionInitialization(Argc, Argv);
 
   if ((IS_RUN_STATUS(STATUS_FAILED)) || (IS_RUN_STATUS(STATUS_DEAD))) {
